packages/framework/src/Model/Customer/User/Role/CustomerUserRoleGroup.php

```diff
@@ -51,8 +48,24 @@ class CustomerUserRoleGroup extends AbstractTranslatableEntity
     public function __construct(CustomerUserRoleGroupData $customerUserRoleGroupData)
     {
         $this->translations = new ArrayCollection();
-        $this->roles = $customerUserRoleGroupData->roles;
         $this->uuid = $customerUserRoleGroupData->uuid ?: Uuid::uuid4()->toString();
+        $this->setData($customerUserRoleGroupData);
+    }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Customer\User\Role\CustomerUserRoleGroupData $administratorRoleGroupData
+     */
+    public function edit(CustomerUserRoleGroupData $administratorRoleGroupData): void
```

Comment:
Probably typo. It should be `$customerUserRoleGroupData`

Reply:
:+1: fixed in https://github.com/shopsys/shopsys/pull/3323/commits/1b3d0b8c0cd06098e652f49d9f10265a50f855f8